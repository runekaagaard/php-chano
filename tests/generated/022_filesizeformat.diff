001- 10.0 KB
001+ 10.0 KiB