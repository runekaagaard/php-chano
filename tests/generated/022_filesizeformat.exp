10.0 KB