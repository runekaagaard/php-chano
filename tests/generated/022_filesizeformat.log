
---- EXPECTED OUTPUT
10.0 KB
---- ACTUAL OUTPUT
10.0 KiB
---- FAILED
