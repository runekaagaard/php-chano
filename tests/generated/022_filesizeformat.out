10.0 KiB